src/Module.zig

```diff
@@ -1848,6 +1848,17 @@ pub const File = struct {
         if (amt != stat.size)
             return error.UnexpectedEndOfFile;
 
+        const resolved_path = try file.fullPath(gpa);
+        defer gpa.free(resolved_path);
+        const real_path = try std.fs.realpathAlloc(gpa, resolved_path);
+        defer gpa.free(real_path);
+        if (!mem.endsWith(u8, real_path, file.sub_file_path)) {
+            log.warn("real path {s} != imported path {s}", .{
+                real_path, resolved_path,
+            });
+            return error.ErrorImportPathMismatch;
```

Comment:
Redundant "error"
```suggestion
            return error.ImportPathMismatch;
```

Reply:
Thx for pointing this out. I blindly copied it over from the C enum.